Base/usr/share/man/man2/mount.md

```diff
@@ -37,14 +38,21 @@ mounted file system.
 
 ### Bind mounts
 
-If `MS_BIND` is specified in `flags`, `fs_type` is ignored and a bind mount is performed
-instead. In this case `source` is treated as a path to a file or directory whose contents
-are overlayed over `target`. This can be used  as an alternative to symlinks or hardlinks.
+If `MS_BIND` is specified in `flags`, `fs_type` is ignored and a bind mount is
+performed instead. In this case `source_fd` is treated as a file descriptor to a
+file or directory whose contents are overlayed over `target`. This can be used
+as an alternative to symlinks or hardlinks.
 
 ## Errors
 
+* `EFAULT`: The `fs_type` or `target` are invalid strings.
 * `EPERM`: The current process does not have superuser privileges.
-* `ENODEV`: The `fs_type` is unrecognized, or the device is not found, or the device doesn't contain a valid filesystem image.
+* `ENODEV`: The `fs_type` is unrecognized, or the file descriptor to source is
+  not found, or the source doesn't contain a valid filesystem image. Also, this
+  error occurs if `fs_type` is valid, but the file descriptor from `source_fd`
+  is not seekable.
+* `EBADF`: The `fs_type` is specified for non-psuedo filesystem, or if `MS_BIND`
+  is specified in `flags`, and `source_fd` is not valid.
```

Comment:
This reads like `!pseudo || (bind && !valid)`, while we need to make it read as `(!pseudo || bind) && !valid`.

How about: "if the `source_fd` is not valid, and either `fs_type` specifies a file-backed filesystem (and not a pseudo filesystem), or `MS_BIND` is specified in `flags`.